docs/locale/fr/LC_MESSAGES/userguide/new_organization.po

```diff
@@ -6,162 +6,168 @@ msgid ""
+"trial period."
 msgstr ""
-"Le mot de passe est utilisé pour chiffrer vos clés privées d'utilisateur et "
-"de l'appareil sur votre ordinateur."
+"Votre organisation et les données associées seront supprimées à la fin de la "
+"période d'essai."
 
-#: ../../userguide/new_organization.rst:48
+#: ../../userguide/new_organization.rst:60
+msgid "Select ``Try Parsec for 15 days``."
+msgstr "Sélectionnez ``Essayez Parsec pendant 15 jours``."
+
+#: ../../userguide/new_organization.rst:61
+msgid "Enter your name and email address."
+msgstr "Saisissez votre nom et votre adresse électronique."
+
+#: ../../userguide/new_organization.rst:67
+msgid "Create an organization on my own Parsec server"
+msgstr "Créer une organisation sur mon propre serveur Parsec"
+
+#: ../../userguide/new_organization.rst:69
 msgid ""
-"Once this process is completed, you can log-in to the organization you just "
-"created."
+"Follow the steps below to create an organization on your own Parsec server:"
 msgstr ""
-"Une fois ce processus complété, vous pouvez vous connecter à l'organisation "
-"que vous venez de créer."
+"Suivez les étapes ci-dessous pour créer une organisation sur votre propre "
+"serveur Parsec :"
 
-#: ../../userguide/new_organization.rst:52
+#: ../../userguide/new_organization.rst:72
+msgid "Select ``Use a different Parsec server``."
+msgstr "Sélectionnez ``Utiliser un autre serveur Parsec``."
+
+#: ../../userguide/new_organization.rst:73
 msgid ""
-"The keys stored encrypted on your computer are what makes Parsec secure. "
-"Hence there is no way to recover if you lost them (e.g. if you computer is "
-"stolen) or if you forget the password protecting them."
+"Specify the organization name and the address of your own Parsec server."
 msgstr ""
-"Les clés stockées sur votre ordinateur assurent la sécurité de Parsec. De ce "
-"fait il n'est pas possible de les récupérer si vous les perdez (par exemple "
-"si votre ordinateur est volé) ou si vous oubliez le mot de passe les "
-"protégeant."
+"Indiquez le nom de l'organisation et l'adresse de votre propre serveur "
+"Parsec."
 
-#: ../../userguide/new_organization.rst:56
+#: ../../userguide/new_organization.rst:79
 msgid ""
-"To avoid losing access to your data you should always have :ref:`multiple "
-"devices <doc_userguide_new_device>` or :ref:`share your workspaces "
-"<doc_userguide_share_data>` with other users."
+"You can read more about hosting Parsec server in the :ref:`Hosting Guide "
+"<doc_adminguide_hosting>`."
 msgstr ""
-"Pour éviter de perdre l'accès à vos données, vous devriez toujours avoir :"
-"ref:`plusieurs appareils <doc_userguide_new_device>` ou bien :ref:`partager "
-"vos espaces de travail <doc_userguide_share_data>` avec d'autre utilisateurs."
+"Vous pouvez en savoir plus sur l'hébergement du serveur Parsec dans le :ref:"
+"`Hosting Guide <doc_adminguide_hosting>`."
+
+#~ msgid "|first_steps_note|"
+#~ msgstr "|first_steps_note|"
```

Comment:
To remove

Reply:
Good catch, I've removed it yesterday but then I saw :sweat_smile:  
![image](https://github.com/user-attachments/assets/f1ff3766-80de-469e-9706-f346b65ed3cb)

Reply:
[`c4bacb9`](https://github.com/Scille/parsec-cloud/pull/8387/commits/c4bacb987ae4861b044d3d002a05a61cc93dd755)